### DISCUSSION TITLE
Welcome to hass-weatherflow2mqtt Discussions!

### DISCUSSION THREAD

**Discussion by briis** [briis/hass-weatherflow2mqtt#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PhillySports** [comment]
Good Afternoon,

Briis or anyone else, anyone willing to help out an extreme NOOB?  I am brand new to Home Assistant from Smartthings.  I have Home Assistant running on the Home Assistant Blue.  I have the Portainer add-on running, but I have never used Docker before or had any other need to use Docker (or Portainer) prior to trying to get my Weatherflow data into Home Assistant, so I am not sure what the next steps are or really where to get started.  I sincerely appreciate any time anyone is willing to share to get this going.

**Reply by briis** [reply]
Now that you use a HA Blue, you have what you need from a HW and base SW view. Let me see if I can write up a detailed guide during the weekend, to get you running using the the UI on a Blue. Stay tuned..

**Reply by PhillySports** [reply]
Thank you very much! Your help (and patience) is greatly appreciated.

**Reply by PhillySports** [reply]
I also just noticed your two other Weatherflow integrations.  Are these just different methods of accomplishing the same goal (getting Weatherflow data into HA) or do they each have different functionality?

**Reply by briis** [reply]
Yes, but note that the UDP version is no longer maintained by me.
If you want to get started quickly, you can go here: https://github.com/briis/smartweather which pulls data directly from WeatherFlow. Before you do that install HACS first, which is a Integration store for all the stuff that is not part of the standard Home Assistant. Read here for more: https://hacs.xyz/

**Reply by PhillySports** [reply]
OK.  Perhaps, I will just wait (if you don't mind and are still willing to help) for your instructions on this one -- since it is still being maintained?  TIA!

**Reply by briis** [reply]
A user just wrote about an issue with HA Blue and manually adding a Container using Portainer. It does not work after the latest update of Portainer. See more here: https://community.home-assistant.io/t/portainer-v2-6-2-unable-to-start-containers/332356/35

So I am not sure how to do this at present. I have a Blue for Testing, so I will play a bit with it to see if I can make it happen.

**Reply by PhillySports** [reply]
OK.  Great.  I appreciate all your time and effort.  In the meantime, maybe I will try to get one of the other older integrations working.
